drivers/src/mldsa87.rs

```diff
@@ -154,15 +154,6 @@ impl Mldsa87 {
     ) -> CaliptraResult<Mldsa87PubKey> {
         let mldsa = self.mldsa87.regs_mut();
 
-        // Wait for hardware ready
-        Mldsa87::wait(mldsa, || mldsa.mldsa_status().read().ready())?;
```

Comment:
We still need the first wait for ready here

Reply:
Thanks! Messed up on the conflict resolution.